# Generated by Django 5.0.2 on 2024-03-02 10:08

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Customers',
            fields=[
                ('customer_id', models.CharField(max_length=9, primary_key=True, serialize=False)),
                ('first_name', models.CharField(max_length=16)),
                ('last_name', models.CharField(max_length=256)),
                ('email', models.CharField(max_length=32)),
                ('phone_number', models.CharField(max_length=50)),
                ('address', models.CharField(max_length=50)),
                ('card_type', models.CharField(max_length=50)),
            ],
            options={
                'db_table': 'customers',
                'managed': True,
            },
        ),
        migrations.CreateModel(
            name='MyappUsers',
            fields=[
                ('id', models.AutoField(primary_key=True, serialize=False)),
                ('username', models.CharField(max_length=50)),
                ('email', models.CharField(max_length=256)),
                ('salt', models.CharField(max_length=32)),
                ('password1', models.CharField(default='', max_length=200)),
                ('password2', models.CharField(default='', max_length=200)),
                ('password3', models.CharField(default='', max_length=200)),
                ('history', models.IntegerField(default=1)),
                ('last_login_attempt', models.DateTimeField(blank=True, null=True)),
                ('failed_login_attempts', models.IntegerField(default=0)),
            ],
            options={
                'db_table': 'site_users',
                'managed': True,
            },
        ),
    ]
